packages/framework/ini-files/config-specs.ini

```diff
@@ -2257,24 +2257,24 @@ use KOKKOS-ARCH|NO-KOKKOS-ARCH
 use USE-ASAN|NO
 use USE-COMPLEX|NO
 use USE-FPIC|NO
-use USE-MPI|NO
+use USE-MPI|YES
```

Comment:
This is likely what caused the MPI build issue.

Reply:
Yep